qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -87,9 +89,26 @@
             iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -i 0 -i 1 -b iozone.xls -f ${disk_letter}:\testfile"
             iozone_timeout = 7200
             wait_bg_time = 180
+        - with_viorng:
+            no no_virtio_rng
+            driver_name = viorng
+            device_name = "VirtIO RNG Device"
+            sub_test = rng_bat
+            target_process = random\w*.exe
+            session_cmd_timeout = 240
+            read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
+            rng_data_rex = "0x\w"
+            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
     variants:
         - uninstall_install:
             need_uninstall = yes
         - downgrade_upgrade:
             need_uninstall = no
             cdrom_virtio_downgrade = isos/windows/virtio-win-downgrade.iso
+    variants:
+        - before_bg_test:
+            run_bg_flag = "before_bg_test"
+        - during_bg_test:
+            run_bg_flag = "during_bg_test"
```

Comment:
Need ignore error during driver test.


15:39:00 ERROR| ShellProcessTerminatedError: Shell process terminated while waiting for command to complete: 'echo list disk > disk && echo exit >> disk && diskpart /s disk'    (status: 1,    output: '\nMicrosoft DiskPart version 6.3.9600\n\nCopyright (C) 1999-2013 Microsoft Corporation.\nOn computer: WIN-IE24DTPILE3\n\nC:\\>Ncat: Connection reset by peer.\n')